src/riak_cs_riak_client.erl

```diff
@@ -113,6 +114,36 @@ pbc_pool_name(undefined) ->
 pbc_pool_name(BagId) when is_binary(BagId) ->
     list_to_atom(lists:flatten(io_lib:format("pbc_pool_~s", [BagId]))).
 
+%% @doc Make a thunk that looks up samples for a given bucket+prefix.
+-spec rts_puller(riak_client(), binary(), iolist(), riak_cs_stats:key()) -> fun().
+rts_puller(RcPid, Bucket, Postfix, StatsKey) ->
```

Comment:
I do have neither strong opinion nor enough knowledge. ... so consulted google-sensei and found the page [1]
In short, they are almost inter-changable but second answer said suffix is used as noun and postfix is as adjective. This makes sense for me. Then, the issue is what `Postfix` here means, whether short for "postfix token" or "suffix". After at length to here, I don't care which to be used and respect your choice :smile: 

[1] http://english.stackexchange.com/questions/81263/postfix-or-suffix